tchannel-benchmark/src/main/java/com/uber/tchannel/benchmarks/PingPongServerBenchmark.java

```diff
@@ -83,9 +89,6 @@ public void benchmark() throws Exception {
                 request,
                 Pong.class
         );
-
-        f.get();
```

Comment:
The strange part is even without f.get I can't get the performance to upto what I wanted. This suggest that there is a performance bottleneck in just making the calls.

Also while logging stuff on request router side, there were performance bottlenecks over there as well but this benchmark is not able to expose that part.